.travis.yml

```diff
@@ -23,7 +24,7 @@ before_install:
   - cd WcaOnRails/
 before_script:
   - bundle exec rake db:reset RAILS_ENV=test
-sudo: false
+sudo: required
```

Comment:
Strange, I guess this documentation is out of date: https://docs.travis-ci.com/user/database-setup/#MySQL-5.6. It says "The recommended way to get MySQL 5.6 is switching to our Trusty CI Environment."